### ISSUE TITLE
Error report

### ISSUE THREAD

**Issue by devmanning** [tabulapdf/tabula#302]

**Comment by jeremybmerrill** [comment]
What error did you get? What were you trying to do? What happened that was not expected?

**Comment by devmanning** [comment]
Hi Jeremy,Thanks for your response. I loaded the pdf file and selected the table to extract and, it gave me the error that it was unable to complete the request. I closed the application and redid the same process and it allowed me the second time. I am sorry unfortunately, I did not copy the error message, I thought while I clicked the button to report the error it would automatically send it.This is a a great tool made my life so much easier. Thanks,Dev 


     On Tuesday, 14 April 2015 12:12 PM, Jeremy B. Merrill <notifications@github.com> wrote:
   

 What error did you get? What were you trying to do? What happened that was not expected?—
